streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1180,11 +1179,15 @@ public String name() {
             public void apply(final StreamTask task) {
                 try {
                     task.suspend();
+                } catch (final CommitFailedException e) {
+                    // commit failed during suspension. Just log it.
+                    log.warn("{} Failed to commit task {} state when suspending due to CommitFailedException", logPrefix, task.id);
                 } catch (final Exception e) {
+                    log.error("{} Suspending task {} failed due to the following error:", logPrefix, task.id, e);
                     try {
                         task.close(false);
                     } catch (final Exception f) {
-                        log.error("{} Closing task {} failed: ", logPrefix, task.id, f);
+                        log.error("{} Closing task {} due to suspension error failed again due to the following error:", logPrefix, task.id, f);
```

Comment:
I was stating that the close call was triggered because suspend call before has an error; and the close call failed again. Will try to reword it a bit.